### ISSUE TITLE
Error getting events when running `docker run` in OpenShift (but the Pod starts successfully)

### ISSUE THREAD

**Issue by l0rd** [joyrex2001/kubedock#33]
### Problem Description

I am running kubedock in a container ([this is the dockerfile](https://github.com/l0rd/micronaut-sample/blob/main/Dockerfile.dev)) on OpenShift using the following command:

```bash
kubedock server --port-forward
```

and when I run 

```bash
export DOCKER_HOST='tcp://127.0.0.1:2475'
docker run --name httpd -d -p 8080:8080 python python -m http.server 8080
```

I get the following error:

```bash
ERRO[0000] error getting events from daemon: Error response from daemon: 404 page not found 
```

But the Pod is created successfully (c.f. screenshots)

### How to reproduce it

The issue can be reproduced on OpenShift Developer Sandbox (a Red Hat Developer account is required but it's free) using the following link: https://workspaces.openshift.com/f?url=https://github.com/l0rd/micronaut-sample

### Screenshots

<img width="820" alt="image" src="https://user-images.githubusercontent.com/606959/232911516-387ff7d4-7957-4e8c-8a48-04bf4e38970d.png">

<img width="974" alt="image" src="https://user-images.githubusercontent.com/606959/232911577-af525b81-674f-4d1c-87f4-51b355736db8.png">

**Comment by joyrex2001** [comment]
The events API is not mandatory for the testcontainers implementation. The docker-cli uses it to wait until the container is started, and will only return when it received the 'start' event to give a better user-experience. Because of the docker-cli wouldn't continue if I returned a not-implemented response, I decided to return a 404 instead. The latter would let the docker-cli continue, but with the error message as above. In https://github.com/joyrex2001/kubedock/commit/fa49ecb0536457899e8b35fc6d47da15c4722220 I added a basic implementation of the events end-point, focussing on just the container lifecycle. This has fixed the error message.

The events end-point also supports a 'filter' argument to only return events that are applicable to the specified filter, which I didn't implement. I think  supporting an id-filter would make sense, but I also think it will not affect the user-experience.
